images/nested-libvirt/Dockerfile

```diff
@@ -15,6 +15,6 @@ RUN yum install -y \
     gettext \
     google-cloud-sdk \
     openssh-clients && \
```

Comment:
The `&&` needs to go away here, now that you're dropping the following `yum`.

/lgtm cancel

Reply:
@wking @sallyom first `yum` is used to install the `epel-release` repo also so if you do something like following would fail because this yum process check the `<package_which_comes_from_epel_release>` with current configured repo and not able to get it.

```
$ yum install -y epel-release <package_which_comes_from_epel_release> 
```

Try something following and you will hit the same.

```console
$ cat Dockerfile 
FROM centos:7
RUN yum install -y \
    epel-release \
    gettext \
    nss_wrapper && \
    yum -y update && \
    yum clean all

$ sudo podman build -t test:test .
[...]
STEP 2: RUN yum install -y     epel-release     gettext     nss_wrapper &&     yum -y update &&     yum clean all
Loaded plugins: fastestmirror, ovl
Determining fastest mirrors
 * base: repo1.ash.innoscale.net
 * extras: repo1.ash.innoscale.net
 * updates: repo1.ash.innoscale.net
base                                                     | 3.6 kB     00:00     
extras                                                   | 3.4 kB     00:00     
updates                                                  | 3.4 kB     00:00     
(1/4): base/7/x86_64/group_gz                              | 166 kB   00:00     
(2/4): extras/7/x86_64/primary_db                          | 187 kB   00:00     
(3/4): updates/7/x86_64/primary_db                         | 3.3 MB   00:00     
(4/4): base/7/x86_64/primary_db                            | 6.0 MB   00:00     
No package nss_wrapper available.   => since it is not present in the configured repo.
```

Reply:
you are correct. :+1: 
sorry for the holdup!!

@wking, I needed the separate yum install -y nss_wrapper afterall -- my bad!! (nss_wrapper needs epel-release, so that has to be installed separately, first) please re-lgtm if all makes sense, thanks!!

Reply:
@praveenkumar I thought the same, but then realized it's the extra &&, causing the docker build to try to run the command `nss_wrapper`  what a mess.. I've removed the extra &&, this is ready to merge if/when CI goes green.  Thanks